src/Epic/SignIn.spec.js

```diff
@@ -2,88 +2,108 @@ import * as epic from './SignIn'
 import * as reducer from './../Redux/State/SignIn'
 import * as session from './../Redux/State/Session'
 import { ActionsObservable } from 'redux-observable'
+  
+describe('Epic :: SignIn :: loadScriptEpic', () => {
+  it('dispatches INITIALIZED action', () => {
+    const action$ = ActionsObservable.of(reducer.initialize())
+    let importedScript = null
+    let initializerObj = null
 
-describe('Epic :: SignIn :: signInEpic', () => {
-  it('dispatches SIGN_IN_SUCCESS action', done => {
+    const deps = {
+      window: {
+        document: { 
+          createElement: () => ({}),
+          querySelector: () => ({
+            appendChild: script => { 
+              importedScript = script
+              script.onload() 
+            },
+          }),
+        },
+        google: { accounts: { id: { 
+          initialize: initObj => { initializerObj = initObj },
+        }}},
+      },
+    }
+
+    return epic.loadScriptEpic(action$, null, deps)
+      .toPromise(Promise)
+      .then(action => {
+        expect(action.type).toEqual(reducer.INITIALIZED)
+        expect(importedScript.src).toEqual('https://accounts.google.com/gsi/client')
+        expect(initializerObj.client_id).toEqual(process.env.REACT_APP_GOOGLE_CLIENT_ID)
+      })
+  })
+})
+
+describe('Epic :: SignIn :: renderSignInButtonEpic', () => {
+  it('render sign in button', () => {
     const action$ = ActionsObservable.of(reducer.signInButtonMounted())
-    const renderMock = [];
+    let elementId = null;
+    let buttonDefinition = {};
 
     const deps = {
-      getGoogleApi: () => ({
-        signin2: {
-          render: (id, options) => {
-            renderMock.push(id)
-            return options.onsuccess('a GoogleUser type')
-          }
-        }
-      })
+      window: { 
+        google: { accounts: { id: { renderButton: (id, definition) => {
+          elementId = id
+          buttonDefinition = definition
+        }}}},
```

Comment:
You can improve the indentation here

Reply:
I wrote it like that because I just need to define `google.accounts.id.renderButton`, so it saves some lines to do it like this